README.md

```diff
@@ -92,4 +92,12 @@ export MOCK_GS1_SERVICE_API_KEY=test456 # Mock GS1 service API key
 # Run seeding scripts
 ./seeding/idr-data.sh
 ./seeding/mock-gs1-data.sh
+```
+
+## End-to-end testing
+```bash
+# Run end-to-end testing scripts
+docker-compose up -d
```

Comment:
I think this command will trigger the `docker-compose.yml` file instead of the testing Docker Compose file (docker-compose.e2e.yml). Also, the docker-compose.e2e.yml file includes a `seeding-data` service that requires the `SEEDING` flag to be set to true for it to run.

Could you double check this, please?